File: mgm-gcp/src/main/java/uk/co/cerihughes/mgm/translate/PlaylistTranslation.java
package uk.co.cerihughes.mgm.translate;

import uk.co.cerihughes.mgm.model.interim.InterimPlaylist;
import uk.co.cerihughes.mgm.model.output.PlaylistApiModel;

public interface PlaylistTranslation extends EntityTranslation {
    PlaylistApiModel translate(InterimPlaylist interimPlaylist);
}

File: mgm-gcp/src/main/java/uk/co/cerihughes/mgm/translate/PlaylistTranslation.kt
package uk.co.cerihughes.mgm.translate

import uk.co.cerihughes.mgm.model.interim.InterimPlaylist
import uk.co.cerihughes.mgm.model.output.PlaylistApiModel

interface PlaylistTranslation : EntityTranslation {
    fun translate(interimPlaylist: InterimPlaylist): PlaylistApiModel?
}